_posts/2024-05-23-pyos-call-for-editors.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: single
+title: "Volunteers needed: pyOpenSci Editorial Board"
+excerpt: "pyOpenSci is looking for volunteers from a wide array of scientific backgrounds to join our Editorial Board."
+author: "Jesse Mostipak"
+permalink: /blog/pyos-call-for-editors-may-2024.html
+header:
+  overlay_image: images/blog/headers/23-05-2024-call-for-editors.png
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## Join the pyOpenSci Editorial Board
+Thanks to the efforts of our incredible team of volunteers, pyOpenSci is growing! As a result, we're currently seeing a large number of package submissions to [our open peer review process](https://www.pyopensci.org/about-peer-review/index.html). In order to continue supporting scientists in the development of open source scientific software, we're looking to add volunteers to our Editorial Board.
+
+The pyOpenSci Editorial Board is comprised of a diverse group of volunteers, with each editor being responsible for the following tasks:
+
+* Finding reviewers from diverse backgrounds who have a mixture of scientific domain and Python experience.
+* Overseeing the entire review process for a package ensuring it runs in a timely and efficient manner.
+* Supporting the submitting authors and reviewers in answering questions related to the review.
+* Determining whether that package should be accepted into the pyOpenSci ecosystem once the review has wrapped up.
+
+It's critical to our mission that our Editorial Board have a combined expertise in various scientific domains, technical expertise in Python packaging, awareness of the importance of documentation in package usability, and awareness of the importance of CI/test suites in order to ensure robust software development. We do not expect any single editor to be an expert in all of these areas!
```

Comment:
```suggestion
It's critical to our mission that our Editorial Board have a combined expertise in various scientific domains, technical expertise in Python packaging, awareness of the importance of documentation in package usability, and awareness of the importance of CI/test suites in order to ensure robust software development. 

IMPORTANT: We do not expect any single editor to be an expert in all of these areas!
```

because people can be intimidated by this role but 9/10 times they are more than capable - like over the top capable to do the role. so i want to make sure that people know - they don't have to be expert in all of the technical stuff to run a great review.